substrate/primitives/runtime/src/traits/mod.rs

```diff
@@ -1371,7 +1415,7 @@ pub trait ExtrinsicMetadata {
 	const VERSION: u8;
 
 	/// Signed extensions attached to this `Extrinsic`.
-	type SignedExtensions: SignedExtension;
+	type SignedExtensions;
```

Comment:
Weird that we did not renamed it here :P

Reply:
I thought that there's no reason to break this metadata trait right now if we don't update the metadata version. Should we rename it now?

Reply:
The trait doesn't define the Metadata format.

Reply:
Ok let's rename it here then.